modules/timeplus/src/main/java/org/testcontainers/containers/TimeplusContainer.java

```diff
@@ -0,0 +1,121 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.ComparableVersion;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Testcontainers implementation for timeplus.
+ */
+public class TimeplusContainer extends JdbcDatabaseContainer<TimeplusContainer> {
+
+    public static final String NAME = "timeplus";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("ghcr.io/timeplus-io/proton:latest");
```

Comment:
we have image name everywhere, may be we can define in one place use it elsewhere